ci/scripts/run-fpga-cw310-tests.sh

```diff
@@ -40,6 +40,10 @@ export BITSTREAM="--offline --list ${SHA}"
 # in case we've crashed the UART handler on the CW310's SAM3U
 trap 'ci/bazelisk.sh run //sw/host/opentitantool -- --interface=cw310 fpga reset' EXIT
 
+# In case tests update OTP or otherwise leave state on the FPGA we should start
+# by loading a bitstream to clear state
+ci/bazelisk.sh run //sw/host/opentitantool -- --interface=cw310 fpga load-bitstream "${BIT_CACHE_DIR}/${BIT_NAME_PREFIX}.orig"
```

Comment:
Dan and I are pretty confident about this. The concern is that if the test fails, then a PR that's running this test will fail, but it will also not update the bitstream, so a subsequent PR with the same bitstream might fail its mask rom tests because of the updated security version.

This change protects all PRs with it from the state a previous PR leaves it's FPGA in (rebooting the SAM3 doesn't invalidate the bitstream on it's own)